lib/use_case/health_check.rb

```diff
@@ -1,19 +1,22 @@
 module UseCase
   class HealthCheck
-    def initialize(route53_gateway: Gateways::Route53.new)
+    def initialize(route53_gateway: Gateway::Aws::Route53.new, delayer: Gateway::Delayer.new(wait_time: 5))
```

Comment:
This Gateway::Aws::Route53 dependency had the wrong namespace.
Never caused an error because always used explicitly.